sdkproject/Assets/Mapbox/Unity/MeshGeneration/LayerVisualizers/VectorLayerVisualizer.cs

```diff
@@ -55,6 +81,7 @@ public void SetProperties(VectorSubLayerProperties properties, LayerPerformanceO
 			switch (properties.coreOptions.geometryType)
 			{
 				case VectorPrimitiveType.Point:
+					break;
```

Comment:
This break is not correct, snapterrain modifier is required for both custom & point primitive types.